Sources/ProjectSpec/Breakpoint.swift

```diff
@@ -27,6 +27,23 @@ public struct Breakpoint: Equatable {
         }
     }
 
+    public enum StopOnStyle: Int, Equatable {
+        case `throw`
+        case `catch`
```

Comment:
Where did these values come from?

Reply:
@yonaskolb 
The editing pop-up of an Exception Breakpoint has two single choice properties as below.
- Exception
  - All
  - Objective-C
  - C++
- Break
  - On Throw
  - On Catch